### ISSUE TITLE
Publish a module to npm

### ISSUE THREAD

**Issue by zeke** [nodejs/i18n#92]
Once we've landed #57 and #77, we'll have translated content flowing into this repo in the form of markdown files sorted into subdirectories by node version and language. To make this translated content easier to consume by projects like [nodejs/website-redesign](https://github.com/nodejs/website-redesign), we should export a node module that is easy to install and easy to use. The name could be something like `nodejs-i18n`.

The existing [`electron-i18n`](https://unpkg.com/electron-i18n/) module can be used as a reference:

- Has no runtime dependencies
- Exports a single JSON object
- Exports pre-parsed GitHub-style HTML with [hubdown](https://github.com/electron/hubdown), a collection of [remark](https://ghub.io/remark) plugins.

The module exports an object with the following structure:

```js
i18n.docs[language][path]
```

For example:

```js
i18n.docs['en-US']['/docs/api/browser-window']
```

Each document object looks something like this:

```js
{ 
  locale: 'en-US',
  slug: 'browser-window',
  category: 'api',
  href: '/docs/api/browser-window',
  githubUrl: 'https://github.com/electron/electron/tree/master/docs/api/browser-window.md',
  crowdinFileId: '68',
  title: 'BrowserWindow',
  description: 'Create and control browser windows.',
  html: '...' 
}

**Comment by lukaszewczak** [comment]
Hi, @zeke I like this idea, how can I help with this?

**Comment by zeke** [comment]
@lukaszewczak once we land #99, we'll have all the files in place. If you want to start on this, feel free! Let me know if you need help or guidance on where to start.

**Comment by alexandrtovmach** [comment]
Closing in favor #261
